### PR TITLE
chore(card-translation): added ukrainian card translations

### PR DESCRIPTION
**Pull request by Awilum**

### CODE REVIEW

File: src/translations.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       np: `${encodeHTML(name)}'${apostrophe} गिटहब तथ्याङ्क`,
       el: `Στατιστικά GitHub του ${encodeHTML(name)}`,
       ru: `Статистика GitHub пользователя ${encodeHTML(name)}`,
+      "uk-ua": `Статистика GitHub користувача ${encodeHTML(name)}`,
```
Comment (anuraghazra) [inline src/translations.js:22]: I believe Ukrain's language code is `uk` 🤔

Comment (Awilum) [reply src/translations.js:22]: Screenshot 2021-01-07 at 16 54 56

Comment (anuraghazra) [reply src/translations.js:22]: Ohh okay got it! thanks.

**Comment by anuraghazra** [comment]
Thanks @Awilum :)

**Pull request merged**